src/index.html

```diff
@@ -0,0 +1,24 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <title>Wbpack Exercise</title>
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.4.0/css/all.min.css" integrity="sha512-iecdLmaskl7CVkqkXNQ/ZH/XLlvWZOJyj7Yy7tcenmpD1ypASozpmT/E0iPtmFIB46ZmdtAc9eNBvH0H/ZpiBw==" crossorigin="anonymous" referrerpolicy="no-referrer" >
+</head>
+<body>
+    <main>
+        <div id="divToDoList" class="bg-white">
+            <div class="list-title">
+                <h2>Today's To Do</h2>
+                <i class="fa-solid fa-rotate"></i>
+            </div>
+            <div class="list-input">
+                <input type="text" placeholder="Add to your list...">
+                <i class="fa-regular fa-arrow-turn-down-left"></i>
+            </div>
+            <div id="todo-list"></div>
+        </div>
```

Comment:
- Good work up to this point by setting up webpack correctly.  However, it is advised that in the requirements your list should be a clone of the part of the minimalist project. Your design appears to be slightly different. Kindly adjust that.

![image](https://github.com/rubydevi/todolist-minimalist/assets/98453979/89279d1c-8e83-4fd7-a711-cfd152ef15ab)

### Checkout the Minimalist project

![image](https://github.com/rubydevi/todolist-minimalist/assets/98453979/67d9cc45-09da-4a7c-8854-b4c6ecdc10cb)